### PR TITLE
DEL-329-Access AMQ console while app is running

### PR DESCRIPTION
**Pull request by STEPHINRACHEL**
**What**
Check if the AMQ console is accessible while the app is running

**To verify**
- Checkout this branch
- Run the below command 
```
export WORKLOAD_WEB_APP_IMAGE=quay.io/<replace-with-your-repo>/workload-app
export GRAFANA_DASHBOARD=true
make local/build-deploy
```
- wait for the app to get deployed
- Login to the grafana and verify if the `AMQ Console` panel is present in workload-app dashboard

**TODO**
- Pass through the OAuth proxy to access the AMQ Console URL

### CODE REVIEW

File: pkg/checks/amq_console.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+				kubeconfig = clientcmd.RecommendedHomeFile
+			}
+
+			config, err = clientcmd.BuildConfigFromFlags("", kubeconfig)
+			if err != nil {
+				log.Errorf("Error occured, %v", err)
+				return
+			}
+		}
+	}
```
Comment (wei-lee) [inline pkg/checks/amq_console.go:26]: should create a shared function to avoid duplicated code here.

File: go.sum [hunk 2]
```diff
@@ -1043,6 +1108,26 @@ gopkg.in/yaml.v2 v2.2.2/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
...
+k8s.io/klog/v2 v2.0.0/go.mod h1:PBfzABfn139FHAV07az/IF9Wp1bkk3vpT2XSJ76fSDE=
+k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6/go.mod h1:GRQhZsXIAJ1xR0C9bd8UpWHZ5plfAS9fzPjJuQ6JL3E=
+k8s.io/utils v0.0.0-20200619165400-6e3d28b6ed19 h1:7Nu2dTj82c6IaWvL7hImJzcXoTPz1MsSCH7r+0m6rfo=
+k8s.io/utils v0.0.0-20200619165400-6e3d28b6ed19/go.mod h1:jPW/WVKK9YHAvNhRxK0md/EJ228hCsBRufyofKtW8HA=
+sigs.k8s.io/structured-merge-diff/v3 v3.0.0-20200116222232-67a7b8c61874/go.mod h1:PlARxl6Hbt/+BC80dRLi1qAmnMqwqDg62YvvVkZjemw=
+sigs.k8s.io/structured-merge-diff/v3 v3.0.0 h1:dOmIZBMfhcHS09XZkMyUgkq5trg3/jRyJYFZUiaOp8E=
+sigs.k8s.io/structured-merge-diff/v3 v3.0.0/go.mod h1:PlARxl6Hbt/+BC80dRLi1qAmnMqwqDg62YvvVkZjemw=
+sigs.k8s.io/yaml v1.1.0/go.mod h1:UJmg0vDUVViEyp3mgSv9WPwZCDxu4rQW1olrI1uml+o=
+sigs.k8s.io/yaml v1.2.0 h1:kr/MCeFWJWTwyaHoR9c8EjH9OumOmoF9YGiZd7lFm/Q=
+sigs.k8s.io/yaml v1.2.0/go.mod h1:yfXDCHCao9+ENCvLSE62v9VSji2MKu5jeNfTrofGhJc=
```
Comment (wei-lee) [inline go.sum:1130]: why this file is changed, but no changes to the go.mod file?

Comment (STEPHINRACHEL) [reply go.sum:1130]: I think this file got updated as I rebased with Davide's PR https://github.com/integr8ly/workload-web-app/pull/15/files

**Comment by STEPHINRACHEL** [comment]
Closing this one as the https://github.com/integr8ly/workload-web-app/pull/24 contains the same code.

**Pull request closed without merging**